### DISCUSSION TITLE
Recommended diode for D205

### DISCUSSION THREAD

**Discussion by ejohnson81** [ZuluSCSI/ZuluSCSI-firmware#148] [General]
Just wanted to know if there is a part number (value?) for the diode to add to the board for SCSI termination.

**Comment by PetteriAimonen** [comment]
If you mean the D205 for feeding power into SCSI bus TERMPWR line (for use by host or other device on SCSI bus), any SOD-123 schottky diode that can handle the current is fine. Example part number: MBRX120LF
